### PR TITLE
Update _index.md / Plugins

### PR DESCRIPTION
**Pull request by Sayanta66**
Fixed broken links and updated the links

### CODE REVIEW

File: site/content/extend/getting-started/_index.md [hunk 1]
```diff
@@ -13,16 +13,16 @@ Mattermost can be extended by building and installing plugins, by embedding Matt
 
 Plugins are easy to build, effortless to maintain, and offer powerful new options to add functionality and UI to the server and web/desktop apps in ways not available with any other integration method.
 
-[Read more about plugins.](/extend/plugins/)
+[Read more about plugins.](https://developers.mattermost.com/contribute/server/plugins/)
```
Comment (justinegeffen) [inline site/content/extend/getting-started/_index.md:16]:
```suggestion
[Read more about plugins.](/extend/plugins/)
```

File: site/content/extend/getting-started/_index.md [hunk 2]
```diff
@@ -13,16 +13,16 @@ Mattermost can be extended by building and installing plugins, by embedding Matt
...
 
 ### Embed
 
 Mattermost is a great communication platform but sometimes it's more convenient to send and receive messages from within a non-Mattermost application. With Mattermost's rich RESTful web service API and OAuth 2.0 support it's possible to build your own clients to embed Mattermost within any application of your choice.
 
-[Read more about embedding.](/extend/embed/)
+[Read more about embedding.](https://docs.mattermost.com/integrations/embedding.html)
```
Comment (justinegeffen) [inline site/content/extend/getting-started/_index.md:22]:
```suggestion
[Read more about embedding.](/extend/embed/)
```

**Review by justinegeffen — APPROVED** [review]
Thank you for this PR. I have reverted two changes, as the original links are working as expected and don't need to be updated. :)

**Review by cwarnermm — APPROVED** [review]
Thank you for your docs contribution!

**Pull request merged**